clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java

```diff
@@ -1484,4 +1485,14 @@ public synchronized List<Node> brokers() {
     public synchronized Node broker(int index) {
         return brokers.get(index);
     }
+
+    @Override
+    public void registerMetricForSubscription(KafkaMetric metric) {
+
+    }
+
+    @Override
+    public void unregisterMetricForSubscription(KafkaMetric metric) {
+
```

Comment:
Is there a reason we do not want to do same functionality as like MockProducer and MockConsumer here? If there is then can we please write comments here.

Reply:
oversight on my part